catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -33,7 +33,7 @@
 	// Reflection in testcases needed
 	// http://stackoverflow.com/questions/1615163/modifying-final-fields-in-java?answertab=votes#tab-top
 
-	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.993f);
+	public static final float CURRENT_CATROBAT_LANGUAGE_VERSION = Float.valueOf(0.994f);
```

Comment:
It doesn't need a conversion, there's simply no template remix information for older programs.
There was already a check for languageVersion 0.993 to 0.994 --> I've added a brief comment in ProjectManager.loadProject()